examples/multimodal/vision_language_foundation/clip/convert_external_clip_to_nemo.py

```diff
@@ -168,8 +168,8 @@ def mapping_hf_state_dict(hf_model):
         ".pre_layrnorm.bias": ".preprocess_layernorm.bias",
         ".post_layernorm.weight": ".transformer.final_layernorm.weight",
         ".post_layernorm.bias": ".transformer.final_layernorm.bias",
-        ".backbone.embeddings.position_embedding.weight": ".backbone.position_embeddings",
-        ".language_model.embeddings.position_embedding.weight": ".language_model.embedding.position_embeddings",
+        ".backbone.embeddings.position_embedding.weight": ".backbone.position_embeddings.weight",
```

Comment:
@yaoyu-33 Do we have a test for this conversion in CICD?

Reply:
I have tested it myself, its not in CICD, I was told that someone was working on a fix for this and there was a separate MR for fixing bugs in conversion. I will find out and get back. If its not there I will add it to CICD